Upgrade-Guide/upgrade_to_3.8.md

```diff
@@ -0,0 +1,11 @@
+## Upgrade procedure from Gluster 3.7.x
+
+The steps to uprade to Gluster 3.8 are the same as for upgrading to Gluster
+3.9. Please follow the detailed instructions from [the 3.9 upgrade
+guide](upgrade_to_3.8.md).

```

Comment:
Is this right ? The link points to `upgrade_to_3.8.md` while the text of the link says `the 3.9 upgrade guide`